lib/hpke/include/hpke/signature.h

```diff
@@ -16,6 +16,7 @@ struct Signature
     P521_SHA512,
     Ed25519,
     Ed448,
+    RSA_SHA256,
```

Comment:
The  WCA used sha512WithRSAEncryption, do you need to add explicit support for 512?